internal/configs/parsing_helpers_test.go

```diff
@@ -383,6 +383,46 @@ func TestParseTime(t *testing.T) {
 	}
 }
 
+func TestParseOffset(t *testing.T) {
```

Comment:
These were moved from `pkg/apis/configuration/validation/virtualserver.go`so that they can be reused by `pkg/apis/configuration/validation/virtualserver.go` and `internal/k8s/validation.go`